### PR TITLE
Added a summary and description for HackNotts 2018

### PR DESCRIPTION
**Pull request by JamesMCo**

### CODE REVIEW

File: src/_events/hacknotts-2018/description.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+## What's a hackathon?
+
+A [hackathon](https://en.wikipedia.org/wiki/Hackathon) is a collaborative programming festival. The main goal is to try to create a project in a relatively short amount of time, such as 24 hours. It's a fantastic opportunity to meet new people, learn from mentors, and make cool stuff.
+
+## What's HackNotts?
+
+HackNotts is our annual hackathon, hosted in the School of Computer Science on Jubilee Campus. This year, we'the event will be running over the weekend of the 24th and 25th of November, with the opening presentations starting some time around 11am on the Saturday, and the closing presentation will be early in the evening on the Sunday.
```
Comment (erinosher) [inline src/_events/hacknotts-2018/description.md]: > `we'the`

I think I found a typo ;)

File: src/_events/hacknotts-2018/description.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+## What's HackNotts?
+
+HackNotts is our annual hackathon, hosted in the School of Computer Science on Jubilee Campus. This year, we'the event will be running over the weekend of the 24th and 25th of November, with the opening presentations starting some time around 11am on the Saturday, and the closing presentation will be early in the evening on the Sunday.
+
+## Who can attend?
+
+You're allowed to attend HackNotts if you are:
+
+- A high school student,
```
Comment (erinosher) [inline src/_events/hacknotts-2018/description.md]: This is not confirmed yet. Lots of hoops to jump through.

Comment (erinosher) [reply src/_events/hacknotts-2018/description.md]: Perhaps change to 18 years or older

**Review by erinosher — CHANGES_REQUESTED** [review]
Thanks so much for contributing.
Just a few typos/factual errors. Once you fix them, you're all good ;)

File: src/_events/hacknotts-2018/description.md (hunk 1 above)
Comment (JamesMCo) [reply src/_events/hacknotts-2018/description.md]: Oh dear! I guess that's what you get when you're writing this stuff late at night 🙂

File: src/_events/hacknotts-2018/description.md (hunk 2 above)
Comment (JamesMCo) [reply src/_events/hacknotts-2018/description.md]: That's fair, I was basing my description on the FAQ from a past event, so details have likely changed. If we're not sure yet, would it be better to just remove this line, seeing as it's covered already by the university student lines following it?

Comment (erinosher) [reply src/_events/hacknotts-2018/description.md]: I think 18 on It's own. And then High School, Undergrad, and Postgrad

Comment (JamesMCo) [reply src/_events/hacknotts-2018/description.md]: Sounds good to me!

**Review by erinosher — APPROVED** [review]
Much better

**Pull request merged**